docs/news/posts/2024-10-04-data-access.md

```diff
@@ -0,0 +1,226 @@
+also the science you conduct.
+
+## Dumping process data to disk
+
+_From AiiDA's internal storage to classical file trees_
+
+As you might be aware, AiiDA uses an SQL
+[database](https://aiida.readthedocs.io/projects/aiida-core/en/v2.6.2/topics/storage.html), as well as an internal [file
+repository](https://aiida.readthedocs.io/projects/aiida-core/en/stable/topics/repository.html#repository) [^1] to store
+your data locally [^2].
+Both are optimized towards high performance and the difference between the typical file-system approach most of us are
+familiar with, and AiiDA's machine-readable internal data storage can make it difficult to get your data out of it in an
+easily accessible way.
+
+Therefore, you, the user, is effectively forced to use the `verdi` CLI interface or AiiDA's Python API (e.g. the `QueryBuilder`) to access your data, making
+the transition towards AiiDA more challenging. Thus, to ease this transition, we have added functionality to dump AiiDA
+`Process`es to disk in an intuitive directory structure via:
+
+```shell
+verdi process dump <pk>
+```
+
+The following video shows the result of running the command for a `PwCalculation` that was used to execute the `pw.x`
+executable of Quantum ESPRESSO:
+
+![PwCalculation dump](./_gifs/calculation-dump-white-10fps-2160p.gif)
+
+And for a more complex `PwBandsWorkChain` [^3]:
+
+![PwBandsWorkChain dump](./_gifs/workflow-dump-white-10fps-2160p.gif)
+
+As you can see, the command works for both, individual calculations, as well as nested workflows [^4], resulting in
+the following output directories:
+
+**`tree` on a dumped example `CalcJob`**
+
+```shell
+dump-PwCalculation-54
+├── README.md
+├── inputs
+│  ├── _aiidasubmit.sh
+│  └── aiida.in
+├── outputs
+│  ├── _scheduler-stderr.txt
+│  ├── _scheduler-stdout.txt
+│  ├── aiida.out
+│  └── data-file-schema.xml
+└── node_inputs
+   └── pseudos
+      └── Si
+         └── Si.pbesol-n-rrkjus_psl.1.0.0.UPF
+```
+
+**`tree -d ` on a dumped example `WorkChain`**
+
+```shell
+dump-PwBandsWorkChain-70
+├── 01-relax-PwRelaxWorkChain
+│  ├── 01-iteration_01-PwBaseWorkChain
+│  │  ├── 01-create_kpoints_from_distance
+│  │  │  └── inputs
+│  │  └── 02-iteration_01-PwCalculation
+│  │     ├── inputs
+│  │     ├── node_inputs
+│  │     │  └── pseudos
+│  │     │     └── Si
+│  │     └── outputs
+│  └── 02-iteration_02-PwBaseWorkChain
+│     ├── 01-create_kpoints_from_distance
+│     │  └── inputs
+│     └── 02-iteration_01-PwCalculation
+│        ├── inputs
+│        ├── node_inputs
+│        │  └── pseudos
+│        │     └── Si
+│        └── outputs
+├── 02-seekpath-seekpath_structure_analysis
+│  └── inputs
+├── 03-scf-PwBaseWorkChain
+│  ├── ...
+...
+```
+
+Therefore, after running the command once, you'll have all data involved in the execution of your workflow directly
+accessible, and can explore it with the Unix command line utilities we all know and love [^5].
```

Comment:
```suggestion
accessible, and can explore it as a standard folder [^5].
```
I'm changing this not only to make the sentence easier to read, but also because in principle you are not forced anymore to use a CLI, you can just open it in a file-browser window.

Reply:
Also, "Unix command line utilities we all know and love" a bit biased 😅

Reply:
@edan-bainglass ofc, get out with your Windows machine :D ❤️